### PR TITLE
master

### PR DESCRIPTION
**Pull request by yaronha**
add natural lang demo func

### CODE REVIEW

File: python/natural-lang/function.yaml.template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+metadata:
```
Comment (pavius) [inline python/natural-lang/function.yaml.template]: 1. You set env twice, once as MANIPULATION_KIND with the value from the template and once as TO_LANG with hardcoded language
2. Please use UpperCase for values: targetLang -> TargetLang

File: python/natural-lang/natural-lang.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+def handler(context, event):
+    context.logger.info('This is an NLP example! ')
+
+    # process and correct the text
+    blob = TextBlob(str(event.body))
+    corrected = blob.correct()
+
+    # debug print the text before and after correction
+    context.logger.debug_with("corrected text", corrected=str(corrected), orig=str(blob))
```
Comment (pavius) [inline python/natural-lang/natural-lang.py]: Logs should start with Upper case. Fix below as well

File: python/natural-lang/natural-lang.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+    # calculate sentiments
+    context.logger.info_with("sentiment",
+                             polarity=str(corrected.sentiment.polarity),
+                             subjectivity=str(corrected.sentiment.subjectivity))
+
+    # read target language from environment and return translated text
+    lang = os.getenv('TO_LANG', "he")
```
Comment (pavius) [inline python/natural-lang/natural-lang.py]: Replace `""` with `''`

Comment (yaronha) [reply python/natural-lang/natural-lang.py]: @pavius fixed all the above

**Pull request merged**